libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -913,4 +735,146 @@ bool _IotMqtt_GetNextByte( void * pNetworkConnection,
 void _IotMqtt_CloseNetworkConnection( IotMqttDisconnectReason_t disconnectReason,
                                       _mqttConnection_t * pMqttConnection );
 
+/*----------------- MQTT Serailization /Deserialization Wrapper functions for Shim------------------*/
+
+/**
+ * @brief The wrapper for  @ref _IotMqtt_SerializeConnect
```

Comment:
Please add a bit more details in comments similar to the old serializer functions